src/adapters/bidderFactory.js

```diff
@@ -250,6 +256,17 @@ export function newBidder(spec) {
     }
   });
 
+  function isUnknownBidder(responseBidder, requestBidder, adapterCode) {
+    let currentBidderSettings = !!bidderSettings.get(requestBidder, 'allowUnknownBidderCodes', adapterCode || null);
+    let unknownBiddersList = bidderSettings.get(requestBidder, 'allowedUnknownBidderCodes', adapterCode || null);
+    if (!!responseBidder && adapterManager.getAdapterCode(responseBidder) !== adapterCode && !!requestBidder && requestBidder !== responseBidder) {
+      if (adapterManager.aliasRegistry[responseBidder] || !currentBidderSettings || (isArray(unknownBiddersList) && (unknownBiddersList[0] !== '*' && !unknownBiddersList.includes(responseBidder)))) {
```

Comment:
Why the first condition - if the reponse bidder is an alias, then it is always unknown?